MinecraftClient/Mapping/Material2Tool.cs

```diff
@@ -0,0 +1,706 @@
+﻿using MinecraftClient.Inventory;
+using MinecraftClient.Mapping;
+using System.Collections.Generic;
+
+namespace MinecraftClient.Mapping
+{
+    class Material2Tool
```

Comment:
Would be better to mark this class as `static`.
```suggestion
    static class Material2Tool
```
And change everything within the class to `static` as well.